src/main/java/ch/qos/logback/more/appenders/FluencyLogbackAppender.java

```diff
@@ -56,40 +62,43 @@ public void start() {
     }
 
     @Override
-    protected void append(ILoggingEvent rawData) {
-        String msg;
-        if (layout != null) {
-            msg = layout.doLayout(rawData);
-        } else {
-            msg = rawData.toString();
-        }
+    protected void append(E event) {
+        String msg = new String(encoder.encode(event));
```

Comment:
I guess you can pass byte[] to fluency. so `new String` won't be needed.

Reply:
Good catch. Fixed